src/common/components/repository-row/index.js

```diff
@@ -281,7 +281,7 @@ export class RepositoryRowView extends Component {
 
     const registeredName = snap.storeName;
 
-    const hasBuilt = !!(latestBuild && snap.snapcraftData);
+    const isBuilt = !!(latestBuild && snap.snapcraftData);
```

Comment:
I guess it comes all way down from backend handler that reads snapcraft.yaml from GH: https://github.com/canonical-ols/build.snapcraft.io/blob/master/src/server/handlers/github.js#L173

So it seems it's never `undefined`.